iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -596,6 +596,122 @@ static LogicalResult setRootConfig(
       entryPointFn, fftOp, tileSizes, DispatchLoweringPassPipeline::CPUDefault);
 }
 
+static LogicalResult setX86DefaultParallelTileSizes(
```

Comment:
Name is not very informative... Maybe rename this to `setX86ParallelWorkgroupTileSizes`

Reply:
Maybe name it to `setX86WorkgroupTileSizes`. The tiling sizes are all set in this method. Then we split it into parallel cases and reduction cases in other places.